test/acceptance/steps/select-business-step.js

```diff
@@ -36,8 +36,14 @@ When(/^user is on the livestock page$/, async function () {
 When(/^user check if livestock are listed$/, async function () {
   await selectBusinessPage.livestockList()
 })
-When(/^user choose the livestock to review$/, async function () {
-  await selectBusinessPage.livestockToReview()
+When(/^user choose dairy cattle for review$/, async function () {
```

Comment:
I think this and the sheep one have been mixed up

Reply:
Line 39 has been separated to cover for each review

Reply:
You wrote "user chose dairy cattle for review" and then ran the script for sheep. This is wrong.